src/tabs/borderless.scss

```diff
@@ -28,6 +28,30 @@
     }
   }
 
+  // #region Horizontal tabs
+  &:not(.iui-vertical) {
```

Comment:
Maybe we need to have both and `iui-horizontal` and `iui-vertical` just to be more explicit.

Reply:
Added `.iui-horizontal` in next push.